lib/services/elasticsearch.js

```diff
@@ -746,34 +746,45 @@ class ElasticSearch extends Service {
    *
    * @returns {Promise}
    */
-  createIndex (index) {
+  async createIndex (index) {
     this._assertValidIndexAndCollection(index);
 
-    return this._client.cat.indices({ format: 'json' })
-      .then(({ body }) => {
-        const esIndexes = body.map(({ index: name }) => name);
+    let body;
 
-        for (const esIndex of esIndexes) {
-          const indexName = this._extractIndex(esIndex);
+    try {
+      const res = await this._client.cat.indices({ format: 'json' });
```

Comment:
```suggestion
      { body } = await this._client.cat.indices({ format: 'json' });
```
You can use destructuring assignment

Reply:
Less weird syntax:
```suggestion
      const { body } = (await this._client.cat.indices({ format: 'json' }));
```

Reply:
@Aschen > dumb syntax is dumb :astonished: (I didn't know that :+1: )

@alexandrebouthinon > I can't do that (Dave), I need the body variable outside the try..catch